documentation/RFC/RFC0001-Aure-Profiles.md

```diff
@@ -0,0 +1,61 @@
+---
+RFC: RFC0001
+Author: Azure PowerShell team
+Status:  Draft
+SupercededBy: n/a
+Version: 1.0
+---
+
+# Enable Azure Profiles with Azure PowerShell
+
+Azure comprises several clouds (Public, Azure Stack, Government), each of which comprises several services (APIs) that have different versions.
+
+Writting an application or a script that is compatible with several cloud can quickly become a very complex task as the developer has to identify for each operation to be performed which API version is compatible on each cloud.
```

Comment:
```suggestion
Writing an application or a script that is compatible with several clouds can quickly become a very complex task as the developer has to identify for each operation to be performed which API version is compatible on each cloud. Since most applications and services in Azure require a set of resources from different services, managing the api-versions of each needed service API in a script quickly multiplies this complexity.
```